examples/timeline-variables.html

```diff
@@ -5,18 +5,47 @@
     <script src="../packages/plugin-html-keyboard-response/dist/index.browser.js"></script>
     <script src="../packages/plugin-image-keyboard-response/dist/index.browser.js"></script>
     <script src="../packages/plugin-preload/dist/index.browser.js"></script>
+    <script src="../packages/metadata/dist/index.browser.js"></script>  
     <link rel="stylesheet" href="../packages/jspsych/css/jspsych.css">
   </head>
   <body></body>
   <script>
 
+    const metadata_options = {
```

Comment:
I don't think we should modify this example. Instead, maybe copy and modify?

Reply:
Changed and moved to metadata-timeline-variables.html.